### ISSUE TITLE
TalkBack unable to confirm media selection when stepping through controls

### ISSUE THREAD

**Issue by dcalhoun** [wordpress-mobile/WordPress-Android#19889]
### Expected behavior
I am able to focus the confirmation button within the media selection modal. 

### Actual behavior
I am unable to focus the confirmation button within the media selection modal, I can only select an invisible "Browse for items" button. 

### Steps to reproduce the behavior
1. Open the block editor. 
2. Enable TalkBack. 
3. Add a media block type, e.g., Image block. 
4. Select "Choose from device." 
5. Select an image. 
6. Attempt to swipe left to step towards the "confirm" button. 

<details><summary>Screen recording</summary>

https://github.com/wordpress-mobile/WordPress-Android/assets/438664/a84aec57-0b73-4e27-9e45-939bfa39d7ac

</details>

> [!NOTE]
> A workaround is directly tapping on the "confirm" button instead of stepping to the control via swipes. This approach may not be accessible for some users.
